content/01.abstract.md

```diff
@@ -2,6 +2,13 @@
 
 Open, collaborative research is a powerful paradigm that can immensely strengthen the scientific process by integrating broad and diverse expertise.
 However, traditional research and multi-author writing processes break down at scale.
-New tools and workflows that rely on automation can catch errors and ensure that contributions are fairly tracked in massively collaborative research.
-We present techniques for overcoming challenges of open research, with special emphasis on manuscript writing.
-These include approaches for managing distributed authors and our new software, named Manubot, for automating citation and many other aspects of manuscript building.
+We present a new software tool named Manubot, available at <https://manubot.org>, for open scholarly writing.
+Manubot adopts the contribution workflow used by many large-scale open source software projects to enable collaborative authoring of scholarly manuscripts.
+With Manubot, manuscripts are written in Markdown and stored in a Git repository to precisely track changes over time.
+By hosting manuscript repositories publicly, such as on GitHub, multiple authors can simultaneously propose and review changes.
+Publication with Manubot is continuous.
+When a manuscript's source changes, the rendered outputs are rebuilt and republished to a web page.
+Manubot automates bibliographic tasks by implementing citation by identifier, where users cite persistent identifiers, whose metadata is then retrieved and rendered to a specified style.
+Manubot modernizes publishing to align with the ideals of open science, making it transparent, reproducibile, immediate, versioned, collaborative, and free of charge.
+Yet, fully open manuscript preparation also presents new challenges, which we faced while orchestrating Manubot's first use case: an open review article with {{deep_review_authors}} authors.
+We discuss solutions to these challenges that arise with next-generation scholarly publishing.
```

Comment:
@agitter @slochower @cgreene want to start reviewing my updated abstract?

Is longer than before, with the goal of providing a more concrete description of what the paper is about. I measured 208 words ([PLOS CB](https://journals.plos.org/ploscompbiol/s/submission-guidelines) limit is 300).

Reply:
Cross referencing #115.  We could also add this pull request to the response letter, but then we can't finalize the response letter link in this pull request.

Reply:
We probably should just switch to using a tag that will correspond to the GitHub release. Perhaps `v3.0`?